assignment/scripts/function-practice.js

```diff
@@ -49,41 +53,107 @@ console.log( 'isPositive - should say false', isPositive(-3) );
 
 // 6. Function to return the _last_ item in an array. If the 
 //    array is empty, return `undefined`.
-function getLast( array ) {
-
+function getLast(array) {
+  let lastItem = undefined;
+  if (array.length) {
+    lastItem = array[array.length - 1];
+  }
+  return lastItem;
 }
+console.log('getLast([1, 2, 3, 4])', getLast([1, 2, 3, 4]));
+console.log('getLast([])', getLast([]));
 
 // 7. Function to find a value in an array. Return true if the 
 //    value is found and false otherwise. Use a loop;
 //    DO NOT use Array.includes, Array.indexOf, or Array.find 
-function find( value, array ){
-  
+function find(value, array) {
+  for (let element of array) {
+    if (element === value) {
+      return true;
+    }
+  }
+  return false;
 }
+console.log("find('z', 'abcdefg')", find('z', 'abcdefg'))
```

Comment:
Is this function for an array or a string? Your implementation happens to work for both, but the documentation seems to indicate it is intended for arrays. Should probably use arrays to test it in that case.